docs/appendices/release-notes/5.8.4.rst

```diff
@@ -108,3 +108,6 @@ Fixes
 
 - Fixed an issue which may cause a ``EXPLAIN ANALYZE`` to throw exception due
   to internal concurrent unsafe access.
+
+- Fixed an issue causing a role to loose it's concrete privileges once it gets
+  granted of another role.
```

Comment:
I think it reads better just like:
```suggestion
  granted another role.
```

Reply:
Thanks applied, also added a link to the `GRANT` statement.

Reply:
nice, thx!